core/lib/components/os/balenaos.js

```diff
@@ -141,7 +142,7 @@ module.exports = class BalenaOS {
 		this.network = options.network;
 		this.image = {
 			input: options.image === undefined ? config.get('leviathan.uploads').image : options.image,
-			path: join(config.get('leviathan.downloads'), `image-${id()}`),
+			path: tmp.tmpNameSync(),
```

Comment:
I like this. I haven't checked what exactly the path becomes when you do `tmp.tmpNameSync()` but hopefully it's the same as `config.get('leviathan.downloads')` so we are cleaning it out at the end of the suite using our suite setup.